core/src/main/tests/JdbcPhoneDaoTest.java

```diff
@@ -0,0 +1,101 @@
+import com.es.core.model.phone.Color;
+import com.es.core.model.phone.JdbcPhoneDao;
+import com.es.core.model.phone.Phone;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.jdbc.JdbcTestUtils;
+
+import javax.annotation.Resource;
+import javax.sql.DataSource;
+import java.math.BigDecimal;
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.junit.Assert.assertEquals;
+
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration("classpath:resources/context/test-applicationContext.xml")
+public class JdbcPhoneDaoTest {
+
+    @Resource
+    private JdbcPhoneDao jdbcPhoneDao;
+
+    @Resource
+    private DataSource dataSource;
+
+    @Resource
+    private JdbcTemplate jdbcTemplate;
+
+    @Before
+    public void setUp() {
+        ApplicationContext applicationContext = new ClassPathXmlApplicationContext("/resources/context/test-applicationContext.xml");
+        dataSource = (DataSource) applicationContext.getBean("dataSource");
+        jdbcTemplate = new JdbcTemplate();
+        jdbcTemplate.setDataSource(dataSource);
+    }
+
+    @Test
+    public void notEmptyDataBaseWhenPhoneDaoTestFindAll() {
+        Assert.assertFalse(jdbcPhoneDao.findAll(0, 5).isEmpty());
+    }
+
+    @After
+    public void tearDown() {
+        JdbcTestUtils.dropTables(jdbcTemplate, "colors", "phones");
+    }
+
+    @Test
+    public void whenTestPhoneDaoGet() {
```

Comment:
Here and in other tests: from method name not really clear what is tested. Think of better names for your tests. Good template for tests names that used a lot is <testedClassMethodName>Should<DoSmth/ReturnSmth>When<SomeCondition>.
For example that method could be named like getShouldReturnCorrectPhoneById